backend/README.md

```diff
@@ -8,12 +8,12 @@ The current environment is initialized through the environment variable `ASPNETC
 
 When running the solution locally it applies the configuration setting in the following order;
 
-1. appsettings.json
-2. appsettings.`[environment]`.json
-3. UserSecrets `(if environment=Development)`
-4. Environment Variables
+1. appsettings.json (within /api)
+2. appsettings.`[environment]`.json (within /api)
+3. UserSecrets `(if environment=Development)` (within /api and /dal)
```

Comment:
For some reason User Secrets aren't working reliably within VSCode.  They are only supposed to work when the environment = Development, but I can't get them working at all.

Reply:
Running in containers won't carry over the secret file json from your user account folder. For developing with containers, ENV vars are preferred.